config/locales/page_titles.en.yml

```diff
@@ -69,7 +69,7 @@ en:
     show_case_page: Viewing case details - %{case_number} - Track-a-query
     show_letter_page: Viewing generated letter - %{case_number} - Track-a-query
     stats: Performance Reports - Track-a-query
-    resend_unlock: Resend unlock instructions
+    resend_unlock: Resend unlock instructions -Track-a-query
```

Comment:
I think we need a space after the hyphen

Reply:
Thanks :)